lib/migration.js

```diff
@@ -66,28 +66,6 @@ function mixinMigration(MsSQL) {
     });
   };
 
-  MsSQL.prototype.isActual = function(cb) {
-    var ok = false;
-    var self = this;
-    async.each(Object.keys(this._models), function(model, done) {
-      self.getTableStatus(model, function(err, fields, indexes) {
-        self.alterTable(model, fields, indexes, function(err, needAlter) {
```

Comment:
Please keep `isActual` in mssql, since it's calling `alterTable`. We can improve it in the 2nd round.